src/components/dashboard-widgets/dashboard-table-widget/dashboard-table-widget.component.ts

```diff
@@ -0,0 +1,46 @@
+import {
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    HostListener,
+    Input,
+    OnDestroy,
+    ViewChild
+} from '@angular/core';
+import { DataWidgetConfig } from '../interfaces/data-widget';
+import { DashboardWidgetComponent } from '../../dashboard';
+
+@Component({
+    selector: 'ux-dashboard-table-widget',
+    templateUrl: './dashboard-table-widget.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DashboardTableWidgetComponent implements DataWidgetConfig, OnDestroy {
+    @Input() id: string = '';
+    @Input() name: string = '';
+    @Input() heading: string = '';
+    @Input() fixedMode: boolean = false;
+    @Input() colSpan: number = 1;
+    @Input() rowSpan: number = 1;
+
+    @ViewChild('widget') widget: DashboardWidgetComponent;
+    @Input() header: ReadonlyArray<any> = [];
```

Comment:
I think we only need strings here, also in the data. Users must cast their data to string.